domains/certificates/Query.API/API.IntegrationTests/CertificateIssuingFailingTests.cs

```diff
@@ -46,15 +46,14 @@ public sealed class CertificateIssuingFailingTests :
         this.factory.RabbitMqOptions = rabbitMqContainer.Options;
         registryConnectorFactory.RetryOptions.RegistryTransactionStillProcessingRetryCount = 1;
         registryConnectorFactory.RetryOptions.DefaultFirstLevelRetryCount = 1;
-        registryConnectorFactory.RetryOptions.DefaultSecondLevelRetryCount = 1;
         registryConnectorFactory.RabbitMqOptions = rabbitMqContainer.Options;
         registryConnectorFactory.ProjectOriginOptions = projectOriginStack.Options;
```

Comment:
No more Second Level Retry?

Reply:
Our 2nd level retry will be the shovel plugin, where we manually can move messages from error queues to non-error queues